CHANGELOG.md

```diff
@@ -6,13 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
-- Option `--oneline_errors` for printing errors on one (long) line.
 ### Changed
-- Fixed a bug in calldata encoding - the input was not properly UTF8 encoded.
 ### Removed
 
+## [7.0.1]
+### Changed
+- Updated to [Sophia 7.0.1](https://github.com/aeternity/aesophia/blob/master/CHANGELOG.md#701)
+
 ## [7.0.0]
+### Added
+- Option `--oneline_errors` for printing errors on one (long) line.
 ### Changed
+- Fixed a bug in calldata encoding - the input was not properly UTF8 encoded.
 - Updated to [Sophia 7.0.0](https://github.com/aeternity/aesophia/blob/master/CHANGELOG.md#700---2021-10-20)
```

Comment:
I think this one is wrong, that date does not match 7.0.0 - must have missed it last time...

Reply:
Fixed here c5deaa70d123dde2b5d95b521e37a8375cdb0731